spec/controllers/jobs_controller_spec.rb

```diff
@@ -92,4 +92,18 @@
       expect(Delayed::Job.find(@running.id)).to be
     end
   end
+
+  describe "POST retry_queued" do
+    before do
+      @not_running = Delayed::Job.create(run_at: Time.zone.now - 1.hour)
+      @not_running.update_attribute(:attempts, 1)
+      sign_in users(:jane)
+    end
+
+    it "run the queued job" do
+      expect(Delayed::Job.last.run_at.to_s).not_to eq(Time.zone.now.to_s)
+      Delayed::Job.awaiting_retry.update_all(run_at: Time.zone.now)
```

Comment:
Modifying the database on the model level would be more feasible for a unit test, what you could do here would be something like `post :retry_queued` to test the actual HTTP request.